libraries/stdlib/samples/test/samples/ranges/ranges.kt

```diff
@@ -52,4 +52,34 @@ class Ranges {
         assertTrue(3.14f in range)
         assertFalse(100.1f in range)
     }
+
+    @Sample
+    fun intStep() {
+        val range = 1..6 step 2
+        assertPrints(range, "[1, 3, 5]")
+
+        assertTrue(1 in range)
+        assertFalse(2 in range)
+        assertTrue(3 in range)
```

Comment:
I'm not sure that these assertions add much to the sample, it should be clear from the above printout which elements are in the progression.

Reply:
> I'm not sure that these assertions add much to the sample, it should be clear from the above printout which elements are in the progression.

removed